test/e2e/page-objects/pages/home/nft-list.ts

```diff
@@ -51,8 +60,12 @@ class NftListPage {
     id: string,
     expectedErrorMessage?: string,
   ) {
+    // await this.driver.delay(100000);
```

Comment:
To remove

Reply:
https://github.com/MetaMask/metamask-extension/pull/32020/commits/f020c9b0ec425014204b33361bbfa917e36d1422